lib/system/supports-colors.js

```diff
@@ -31,9 +31,8 @@ module.exports = (function () {
     return false;
   }
 
-  if (argv.indexOf('--color') !== -1 ||
-    argv.indexOf('--color=true') !== -1 ||
-    argv.indexOf('--color=always') !== -1) {
+  if (!argv.indexOf('--no-color') !== -1 ||
+    !argv.indexOf('--color=false') !== -1) {
```

Comment:
Ok thanks, your right I looked below and found this pull https://github.com/Marak/colors.js/pull/154 which fixed it.